lib/modules/syntaxHighlighting.js

```diff
@@ -0,0 +1,20 @@
+/* @flow */
+
+import hljs from 'highlight.js';
+import { Module } from '../core/module';
+import { watchForThings } from '../utils';
+
+export const module: Module<*> = new Module('syntaxHighlighting');
+
+module.moduleName = 'syntaxHighlightingName';
+module.category = 'appearanceCategory';
+module.description = 'syntaxHighlightingDesc';
+module.disabledByDefault = false;
+
+module.beforeLoad = () => {
+	watchForThings(['post', 'comment', 'message'], highlight);
```

Comment:
Instead of `watchForThings(['post'])`, use `watchForElements(['selfText'], null, ele => hljs.highlightElement(ele))` as this will look for code in posts when they become are expanded.